oops/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please don't checkin your local .vscode settings because not everyone wants to use the same settings. I thought I had changed .gitignore to prevent this but apparently what I did wasn't sufficient.